### ISSUE TITLE
Windows are unblurred when webcam is in use (google meet, zoom, etc.)

### ISSUE THREAD

**Issue by forivall** [dwarvesf/blurred#29]
### Describe the bug

When using any app that accesses the webcam (video chat programs), all windows are unblurred. My main use case for Blurred is to focus on meetings while they're happening, so this is a bummer.

### Steps to reproduce

1. Go to meet.google.com
2. Click on New Meeting, select Instant Meeting
3. See all windows are unblurred

### Expected behaviour

Other windows should be blurred.

**Screenshots**

![Screenshot 2023-02-13 at 12 56 30 PM_redact](https://user-images.githubusercontent.com/760204/218574212-57296004-195a-450d-b843-7a403aaf18eb.png)

### Environment

You may write here the specifications like the version of the project, operating system, or hardware if applicable.

**Desktop (please complete the following information):**
 - OS: MacOS 13.1
 - Version 1.2.0 (3)

### Logs / Stack trace

```
Insert your log/stack trace here
```

**Comment by viettrungphan** [comment]
The way the app blurred windows is to insert a transparent black window(blurred) above those windows (accept topmost) however many apps can make themselves topmost, they want to stay floating on top like utility apps, video call, PiP...(system dock is one example, it usually on top of other windows). This case looks like the app trying to do it when you trigger the Instance meeting, they bring all windows to the top.

**Comment by Tolsi** [comment]
it seems that in Mac OS 13 only the desktop is blurred, but not other windows

ps fixed after update to OS X 13.2.1

**Comment by forivall** [comment]
Yeah, it seems like when the screen recording api is invoked, macos messes with window layering and [this code](https://github.com/dwarvesf/blurred/blob/v1.2.0/Blurred/Utilities/DimManager.swift#L89-L119) needs to be updated to handle it.
